ftw/blueprints/sections/multilingual.py

```diff
@@ -1,11 +1,11 @@
 from collective.transmogrifier.interfaces import ISection
 from collective.transmogrifier.interfaces import ISectionBlueprint
 from collective.transmogrifier.utils import defaultMatcher
-from plone.multilingual.interfaces import ILanguage
-from plone.multilingual.interfaces import IMutableTG
-from plone.multilingual.interfaces import ITranslatable
-from plone.multilingual.interfaces import ITranslationManager
+from plone.app.multilingual.interfaces import IMutableTG
```

Comment:
Ok.

I didn't know how you wanted tu support both P4 and P5, that was the
primary reason of the pull-request.

I will work on it (also on the ftw.inflator pull-request)

On Tue, Nov 17, 2015 at 9:42 AM, Jonas Baumann <notifications@github.com>
wrote:

> In ftw/blueprints/sections/multilingual.py
> <https://github.com/4teamwork/ftw.blueprints/pull/23#discussion_r45033597>
> :
>
> > @@ -1,11 +1,11 @@
> >  from collective.transmogrifier.interfaces import ISection
> >  from collective.transmogrifier.interfaces import ISectionBlueprint
> >  from collective.transmogrifier.utils import defaultMatcher
> > -from plone.multilingual.interfaces import ILanguage
> > -from plone.multilingual.interfaces import IMutableTG
> > -from plone.multilingual.interfaces import ITranslatable
> > -from plone.multilingual.interfaces import ITranslationManager
> > +from plone.app.multilingual.interfaces import IMutableTG
>
> Those imports do not work on Plone 4:
>
> ImportError: cannot import name IMutableTG
>
> I think we need to do conditional imports in order to support both Plone
> versions.
> We still have a lot of Plone 4's running and are not yet ready to drop
> Plone 4 support in ftw.blueprints.
>
> Could you change the imports to support both versions?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/4teamwork/ftw.blueprints/pull/23/files#r45033597>.
>



-- 
Mikel Larreategi
mlarreategi@codesyntax.com

CodeSyntax
Azitaingo Industrialdea 3 K
E-20600 Eibar
Tel: (+34) 943 82 17 80